src/main/resources/assets/mekanism/lang/zh_tw.json

```diff
@@ -273,24 +270,24 @@
   "connection.mekanism.normal": "正常",
   "connection.mekanism.pull": "抽出",
   "connection.mekanism.push": "輸入",
-  "constants.mekanism.alpha_warning": "警告:Mekanism當前處于alpha狀態,不建議在模組包中廣泛使用.遊戲可能會崩潰,你可以閱讀更多有關其他問題的訊息 %s.",
+  "constants.mekanism.alpha_warning": "警告:Mekanism當前處於alpha狀態,不建議在模組包中廣泛使用.遊戲可能會崩潰,你可以閱讀更多有關其他問題的訊息 %s.",
   "constants.mekanism.alpha_warning.here": "這裡",
   "constants.mekanism.debug_title": "Mekanism Debug",
   "constants.mekanism.error": "錯誤",
-  "constants.mekanism.forge": "MinecraftForge",
-  "constants.mekanism.ic2": "IndustrialCraft",
-  "constants.mekanism.log_format": "[%s] %s",
-  "constants.mekanism.mod_name": "Mekanism",
+  "constants.mekanism.forge":"MinecraftForge",
+  "constants.mekanism.ic2":"IndustrialCraft",
+  "constants.mekanism.log_format":"[%s] %s",
+  "constants.mekanism.mod_name":"Mekanism",
```

Comment:
Could you please remove changes like this where you are just changing spacing between the key and the translation? It makes it a bit harder to see what is actually changing.